frontend/src/features/memory/components/MemoryTipTapEditor.tsx

```
@@ -62,6 +64,110 @@ export function MemoryTipTapEditor({
 
+  useEffect(() => {
+    if (!editor) return;
+
+    const clearHighlights = () => {
+      const root = editor.view.dom as HTMLElement | null;
+      if (!root) return;
+      root
+        .querySelectorAll<HTMLElement>(
+          ".memory-key-heading, .memory-key-label-line",
+        )
+        .forEach((node) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)